docs/lightsheet/index.md

```diff
@@ -17,6 +17,30 @@ Version 2 has 5 new fields for metadata describing the Z-dimension specifically
 The other fields function the same way for all assays using antibodies. For more information, see the [Antibodies TSV documentation](../antibodies).
 
 ## Directory schemas
+### v1
+
+| pattern | required? | description |
+| --- | --- | --- |
+| <code>Level0/Channel[^/]+/[^/]+\.csv</code> | ✓ | **[QA/QC]** Contains metadata and channel info. |
+| <code>Level0/Channel[^/]+/[^/]+\.czi</code> |  | Zeiss raw image file. |
+| <code>Level0/merged/Channel[^/]+/[^/]+\.czi</code> |  | Merged Zeiss raw image file. |
```

Comment:
I think here and elsewhere where there is a "merged" subdirectory the filenames for the merged images should *not* start with `Channel`. I suggest using `MergedChannels.czi`, `MergedChannels.tif`, etc.

Reply:
Since directory and filenames generally seems to start with an uppercase letter, I would recommend that you change the `merged` subdirectory to `Merged` for all levels.